src/base-element.js

```diff
@@ -482,6 +482,22 @@ export class BaseElement {
   }
 
   /**
+   * Utility method that bubbles events from the given element
```

Comment:
Should this be called `forwardEvents`. I think it is not clear enough what this does. Am I getting it correctly:

- It listens for non-bubbling events
- and transports them to bubbling custom events of the name name

?